message_types.go

```diff
@@ -158,6 +159,12 @@ type FullMessage struct {
 	AttData [][]byte
 }
 
+type FullMessageBuffered struct {
+	Message
+
+	AttData []*bytes.Buffer
+}
```

Comment:
Any reason to have this type in addition to the other one? I think only one of these types will ever be used in practise. Keeping just one type means you could also keep just one client method `c.GetFullMessage(...) (FullMessage, error)`. 

It might be nice to define a type 

```
type AttData []*bytes.Buffer

func (att AttData) ToBytes() [][]byte {
	// call `.Bytes()` on each buffer
}
```

Reply:
I have merged the buffers in the type so both are available.